pom.xml

```diff
@@ -36,6 +36,7 @@
 
 		<docker.image.prefix>snomedinternational</docker.image.prefix>
 		<slf4j.version>1.7.5</slf4j.version>
+		<aws-java-sdk.version>1.11.221</aws-java-sdk.version>
```

Comment:
This explicit version dependency matches the version that was pulled in via `aws-request-signing-apache-interceptor` previously 👍